bindata/network/multus-admission-controller/001-service.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: multus-admission-controller
-  namespace: openshift-multus
+  namespace: {{.AdmissionControllerNamespace}}
```

Comment:
In hypesrshift `AdmissionControllerNamespace` is the[ controllplane namespace in the management cluster](https://github.com/openshift/cluster-network-operator/pull/1516/commits/8283ed93b0da47cc72366ac9d67088abac0e53fb#diff-ca01c219a1218bfb16fb2751022f48388e502868794d249adfad771571f07191R68) but this is being rendered in the hosted cluster since it is missing the annnotation